_posts/2021-08-11-persistent-identifiers.md

```diff
@@ -0,0 +1,84 @@
+---
+title:  "Exposing and resolving persistent identifiers on the web"
+author: Rachel Heaven
+categories:
+  - Open Data
+tags:
+  - Linked Data
+  - RDF
+---
+
+# Exposing and resolving persistent identifiers on the web
+
+## What are persistent identifiers?
+
+Within our internal corporate databases we have well managed persistent identifiers for most of our important data items, in the form of foreign key columns in relational databases. These are either auto-incrementing integers or a short text code.  They uniquely identify a record, can be used to reference it from other database tables are not allowed to be changed. The codes are guaranteed to be unique within that set of data, but may not be unique across BGS's other data resources and certainly not unique outside BGS.
```

Comment:
Is there a missing 'and' - from other database tables AND are not allowed to be changed.

Reply:
fixed